Fix wrong type for matrix size (replace double by size_t).

--- ./include/roboptim/core/derivable-function.hh       2011-10-10 12:11:24.000000000 +0200
+++ ./include/roboptim/core/derivable-function.hh~      2011-10-10 12:11:29.000000000 +0200
@@ -70,7 +70,7 @@
     typedef matrix_t jacobian_t;
 
     /// \brief Jacobian size type (pair of values).
-    typedef std::pair<value_type, value_type> jacobianSize_t;
+    typedef std::pair<size_type, size_type> jacobianSize_t;
 
 
     /// \brief Return the gradient size.
